nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestPutSQL.java

```diff
@@ -144,6 +168,7 @@ public void testInsertWithGeneratedKeys() throws InitializationException, Proces
     @Test
     public void testFailInMiddleWithBadStatement() throws InitializationException, ProcessException, SQLException, IOException {
         final TestRunner runner = TestRunners.newTestRunner(PutSQL.class);
+/****
```

Comment:
I'd suggest to simply remove all commented blocks for better visibility, don't you think ?